Email verification for SSO (#7311)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/sso-overview.md

- **User Exists?**: This step checks if the user already exist in <Constant name="cloud" />'s user database.
  - **Yes**: If so, skip the user creation process
  - **No**: If so, create a new entry in the <Constant name="cloud" /> database for the new user.

> - **Create <Constant name="cloud" /> User**: This will create a new entry in the <Constant name="cloud" /> database for the new user. This user record contains the user's email address, first and last name, and any IdP attributes (e.g. groups) passed along from the Identity Provider. dbt Cloud will send a verification email, and the user needs to follow the steps [here](docs/cloud/manage-access/invite-users#user-experience).

Author: ```suggestion
- **Create <Constant name="cloud" /> User**: This will create a new entry in the <Constant name="cloud" /> database for the new user. This user record contains the user's email address, first and last name, and any IdP attributes (e.g. groups) passed along from the Identity Provider. dbt Cloud will send a verification email, and the user needs to follow the steps [here](/docs/cloud/manage-access/invite-users#user-experience).
```

> - **Create <Constant name="cloud" /> User**: This will create a new entry in the <Constant name="cloud" /> database for the new user. This user record contains the user's email address, first and last name, and any IdP attributes (e.g. groups) passed along from the Identity Provider.  dbt Cloud will send a verification email, and the user needs to follow the steps [here](docs/cloud/manage-access/invite-users#user-experience).

Author: ```suggestion
- **Create <Constant name="cloud" /> User**: This will create a new entry in the <Constant name="cloud" /> database for the new user. This user record contains the user's email address, first and last name, and any IdP attributes (e.g. groups) passed along from the Identity Provider.  dbt Cloud will send a verification email, and the user needs to follow the steps [here](docs/cloud/manage-access/invite-users#user-experience).
```
```suggestion
- **Create <Constant name="cloud" /> User**: This will create a new entry in the <Constant name="cloud" /> database for the new user. This user record contains the user's email address, first and last name, and any IdP attributes (e.g. groups) passed along from the Identity Provider. dbt Cloud will send a verification email, and the user needs to follow the steps [here](docs/cloud/manage-access/invite-users#user-experience).
```

> - **Create <Constant name="cloud" /> User**: This will create a new entry in the <Constant name="cloud" /> database for the new user. This user record contains the user's email address, first and last name, and any IdP attributes (e.g. groups) passed along from the Identity Provider. dbt Cloud will send a verification email, and the user needs to follow the steps [here](/docs/cloud/manage-access/invite-users#user-experience).

Reviewer: ```suggestion
- **Create <Constant name="cloud" /> User**: This will create a new entry in the <Constant name="cloud" /> database for the new user. This user record contains the user's email address, first and last name, and any IdP attributes (for example, groups) passed along from the Identity Provider. dbt Cloud will send a verification email, and the user must follow the steps in the [User experience section](/docs/cloud/manage-access/invite-users#user-experience) to use SSO in <Constant name="cloud" />.
```
